appengine/flexible/api_gateway/gateway/api_gateway.py

```diff
@@ -0,0 +1,46 @@
+import requests
+import sys
+from flask import Flask
+app = Flask(__name__)
+
+#development ports {static: 8003, flask1: 8001, flask2: 8002}
+@app.route('/test')
+def test():
+    return "GATEWAY OPERATIONAL"
+
+@app.route('/')
+def root():
+    res = requests.get('https://static-dot-flask-algo.appspot.com/')
+    return res.content
+
+@app.route('/hello/<service>')
+def say_hello(service):
+    services = {
+        'flask1': { 'url': 'https://flask1-dot-flask-algo.appspot.com/', 'send': False },
```

Comment:
@travisoneill they should be logged under the `stdout` log in the cloud console.

If I understand correctly, we're making some changes to the environment variables. It might be better to just do something like this: 

```python
PROJECT_ID = os.environ.get('GAE_LONG_APP_ID')
DEFAULT_MODULE_URL = '{}.appspot.com'.format(PROJECT_ID)
```